### PR TITLE
feat: replacing WS_BASE_URL with TENANT_NAME and WORKSPACE_ID 

### PR DESCRIPTION
**Pull request by liorfr**
Instead of using WS_BASE_URL we will use 

- TENANT_NAME for subaccount
- WORKSPACE_ID for ws

### CODE REVIEW

File: test/context_manager.spec.ts [hunk 1]
```diff
@@ -8,10 +8,12 @@ describe("Test context manager", () => {
   const extensionNameA = "aaa";
   const extensionNameB = "bbb";
 
-  const USER = "koko@sap.com";
-  const ENVIRONMENT = "prod";
-  const LANDSCAPE = "eu10";
-  const TENANTID = "111-a11-111";
+  const user = "koko@sap.com";
```
Comment (allacherkes) [inline test/context_manager.spec.ts:11]: out of curiosity, why did you change it to small letters?

**Review by allacherkes — APPROVED** [review]

File: test/context_manager.spec.ts (hunk 1 above)
Comment (liorfr) [reply test/context_manager.spec.ts:11]: because it is used in the test (parameter) and not as const
